### PR TITLE
[Tool Request] XXExploiter #2774

### PR DESCRIPTION
**Pull request by dualfade**
please review. Thank you.
~!>D

### CODE REVIEW

File: packages/xxexploiter/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  npm install && \
+    npm run build
+}
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -dm 755 "$pkgdir/usr/share/$pkgname"
+
```
Comment (Edu4rdSHL) [inline packages/xxexploiter/PKGBUILD:35]: Remove blank line.

File: packages/xxexploiter/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
+  install -Dm 644 -t "$pkgdir/usr/share/licenses/$pkgname/" LICENSE.txt
+
+  rm -rf *.md LICENSE
+
```
Comment (Edu4rdSHL) [inline packages/xxexploiter/PKGBUILD:40]: Remove blank line.

File: packages/xxexploiter/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=xxexploiter
+pkgver=81.4a2eb91
+pkgrel=1
+pkgdesc="It generates the XML payloads, and automatically starts a server to serve the needed DTD's or to do data exfiltration."
+arch=('any')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-webapp')
```
Comment (Edu4rdSHL) [inline packages/xxexploiter/PKGBUILD:9]: Add `blackarch-automation` to groups.

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]

File: packages/xxexploiter/PKGBUILD (hunk 3 above)
Comment (dualfade) [reply packages/xxexploiter/PKGBUILD:9]: added

File: packages/xxexploiter/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/xxexploiter/PKGBUILD:35]: adjusted

File: packages/xxexploiter/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/xxexploiter/PKGBUILD:40]: adjusted

Comment (Edu4rdSHL) [reply packages/xxexploiter/PKGBUILD:38]: You removed the wrong line :p

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]

**Pull request merged**